app/models/rook.rb

```diff
@@ -1,2 +1,6 @@
 class Rook < Piece
+  def piece_rules?(to_x:, to_y:)
+    return true if to_y == y_position || to_x == x_position
```

Comment:
Jack, yes that's true... but since a Rook can only move horizontally or vertically, won't either the x coordinate or the y coordinate always remain unchanged?